ls1intum/Artemis
# Created a Word / String Count Service for German/English

### Checklist

### Motivation and Context


Closes #1528

### Description

The goal of this pull request is to implement a word count / character count functionality for text exercises. Both the user when entering an answer and the tutor when correcting an answer should be displayed the word count and character count.


### Steps for Testing


1. Log in to Artemis
2. Navigate to a text exercise as a student and enter some text, the word and character count should change
3. View the submission as a tutor, the word and character count should be displayed!

### Screenshots



![TutorView](https://user-images.githubusercontent.com/29718932/83295415-36fc9500-a1ef-11ea-923f-2db0663cc44e.gif)
![Making a new submission](https://user-images.githubusercontent.com/29718932/83295430-3c59df80-a1ef-11ea-8047-c1af422488af.gif)
